kitsune/questions/jinja2/questions/includes/aaq_macros.html

```diff
@@ -38,25 +40,11 @@ <h3 class="card--title">
   </div>
 {%- endmacro %}
 
-{% macro selected_product(product, can_change) -%}
-  <div id="selected-product">
-    <br />
-    <p>{{ _('Product') }}: <strong><span>{{ product.name }}</span></strong>
-    {% if can_change %}
-      <a class="text-body-sm"
-         href="{{ url('questions.aaq_step1') }}{% if is_mobile_device %}?q=change_product{% endif %}">
```

Comment:
If this is not needed anymore let's also remove the check from the view. Without this we had an infinite loop in the old desgin.

Reply:
I forgot to test this, this logic should be moved into the progress bar to ensure we don't get users stuck in a loop. But it also seems like the auto-selection on mobile is broken, so I'll have to look into that.